src/components/ShareSocialCta/index.js

```diff
@@ -116,3 +116,7 @@ export default function(props) {
     </Location>
   );
 }
+
+function getLocation() {
```

Comment:
This doesn't appear to being called anywhere, seems to be dead code. Can you please remove it? :)